tests/testthat/test-get_param.R

```diff
@@ -0,0 +1,48 @@
+test_that("can use configuration file via options", {
+  config_file <- glue::glue("
+    default:
+      a_field: a_value
+  ")
+  config_path <- tempfile()
+  readr::write_lines(config_file, config_path)
+
+  withr::local_options(list(r2dii_config = config_path))
+  expect_equal(get_param("a_field")(), "a_value")
+})
+
+test_that("outputs the expected value", {
+  config_file <- glue::glue("
+    default:
+      a_field: a_value
+  ")
+  config_path <- tempfile()
+  readr::write_lines(config_file, config_path)
```

Comment:
This may be refactored to remove duplication. I'll leave it as is for now, to maybe refactor is in a peer programming session with @MonikaFu